tests/phpunit/tests/test-install.php

```diff
@@ -16,10 +16,29 @@ function test_activate() {
 		$this->assertEquals( POLYLANG_VERSION, $options['version'] );
 	}
 
+	/**
+	 * This test requires the definition of the constants WP_UNINSTALL_PLUGIN
+	 * The constant PLL_REMOVE_ALL_DATA must not be defined
+	 */
 	function test_no_uninstall() {
```

Comment:
The test looks good to me. Maybe we could now rename it, because it tests the uninstall, but without removing the data.

So maybe: 'test_uninstall_without_removing_data'?